src/main/java/org/ibp/api/rest/inventory/manager/LotResource.java

```diff
@@ -278,5 +280,27 @@ public long getCount() {
 		return new ResponseEntity<>(HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "It will retrieve a lot", notes = "It will retrieve lot by lotUUID")
+	@RequestMapping(value = "/crops/{cropName}/lots/{lotUUID}", method = RequestMethod.GET)
+	@PreAuthorize(HAS_MANAGE_LOTS + " or hasAnyAuthority('VIEW_LOTS')")
+	@ResponseBody
+	@JsonView(InventoryView.LotView.class)
+	public ResponseEntity<ExtendedLotDto> getLot(@PathVariable final String cropName, @PathVariable final String lotUUID) {
+
+		final LotsSearchDto searchDTO = new LotsSearchDto();
+		searchDTO.setLotUUIDs(Arrays.asList(lotUUID));
```

Comment:
@cuenyad  It is ok, but when creating a list for a single element I prefer to use Collections.singletonList
https://stackoverflow.com/questions/26027396/arrays-aslist-vs-collections-singletonlist